game/scripts/vscripts/Sections/Chapter2/SectionCreeps.ts

```diff
@@ -138,10 +141,8 @@ const onStart = (complete: () => void) => {
                         const godzMudGolem = context[CustomNpcKeys.GodzMudGolem]
                         setUnitPacifist(godzMudGolem, true);
                     }),
-                    tg.setCameraTarget(context => context[CustomNpcKeys.GodzMudGolem]),
+                    tg.panCameraLinear(_ => playerHero.GetAbsOrigin(), context => context[CustomNpcKeys.GodzMudGolem].GetAbsOrigin(), 1),
```

Comment:
I think we can remove this pan as the player is already close and we have particles for speakers now to draw attention to them too. Panning camera while not being frozen doesn't feel so good.